src/bin/inx-chronicle/main.rs

```diff
@@ -44,6 +44,26 @@ async fn main() -> eyre::Result<()> {
         ByteSize::b(db.size().await?)
     );
 
+    match db
+        .collection::<ApplicationStateCollection>()
+        .get_last_migration()
+        .await?
+        .as_deref()
+    {
+        None => {
+            db.collection::<ApplicationStateCollection>()
+                .set_last_migration(migrations::LATEST_VERSION)
+                .await?;
+        }
+        Some(migrations::LATEST_VERSION) => (),
```

Comment:
Maybe we should log that everything is up to date.

Reply:
I'd rather only log in the other cases, but if you feel strongly about it I can add it.